README.md

```diff
@@ -1,21 +1,25 @@
 ## BASH Script Storage Project
 
-This is a qucik BASH scripting project that determines how much storage you have on your machine. This is one of the first projects I have worked on while on my jounrey to learn Linux and BASH Scripting.  
+This is a BASH scripting project that determines how much storage you have on your machine. 
+
+## Requirements
+* Bash `vX.X.X`
+* Vim or any bash text editior
 
 ## Figuring our disk space on the CLI (Command Line Interface)
 
-For starters, I am on a Mac using iMac Sierra Version 10.12.6 so some of the commands may differ from machine to machine. Open your CLI and type in the df (disk free) command which will show you the amount of space available on the machine you are using. I would recommend using -H at the end of the command which stands for “human-readable” and this will provie you a general idea of how much space is available on your machine. In this scenario, I will be actually using -k because I want to see the data in kilobytes which will eventually be converted into gigabytes.
+Open your CLI and type in the df (disk free) command, `df` which will show you the amount of space available on the machine you are using. I recommend using `-H` at the end of the command which stands for “human-readable”. This will provie you with a general idea of how much space is available on your machine. In this scenario, I will be actually using `-k` because I want to see the data in kilobytes which will eventually be converted into gigabytes.
```

Comment:
When specifying files, commands, or folders use the backtics as it makes it easier for the reader to see that this is a technical piece.